deps/childprocess.lua

```diff
@@ -164,4 +164,71 @@ local function spawn(command, args, options)
   return em
 end
 
+local function exec(command, args, options, callback)
+  local file
+  local stringArgs = command..table.concat(args, ' ')
```

Comment:
You're right, but:
https://github.com/nodejs/node/blob/master/lib/child_process.js#L101-L107
So when I cycle back to this Ill port both